package.json

```diff
@@ -10,7 +10,7 @@
     "test": "NODE_ENV=test nyc mocha --compilers js:babel-register ./server/tests/ --timeout 5000ms --exit --no-deprecation",
     "start": "npm run build && node dist/index.js",
     "build": "rimraf dist/ && babel ./ -d dist/ --ignore node_modules --copy-files",
-    "start:dev": "nodemon --watch server --exec babel-node ./index.js",
+    "start:dev": "npm run migrate && nodemon --watch server --exec babel-node ./index.js",
```

Comment:
pls revert this. you should run 'test' if u need to migrate or run migrate before you start, so migrate doesn't run every-time dev server starts

Reply:
I have effected the change

Reply:
I have effected the changes